src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -771,6 +784,31 @@ def cosym(self):
                 "Laue group determined by dials.cosym: %s" % best_space_group.info()
             )
 
+    def reindex(self):
+        logger.debug("Running reindexing")
+        logger.info("Re-indexing to reference")
+        experiments_filename = self._data_manager.export_experiments("tmp.expt")
+        reflections_filename = self._data_manager.export_reflections("tmp.refl")
+        reindex = Reindex()
+        auto_logfiler(reindex)
+        reindex.set_experiments_filename(experiments_filename)
+        reindex.set_indexed_filename(reflections_filename)
+        reindex.set_reference_file(self._params.reference)
+
+        if self._params.symmetry.space_group is not None:
```

Comment:
We need to think carefully about what we want to happen if we set a reference file. Reindexing against a reference only make sense if the space groups are the same. So if we supply a reference file, then I think we should extract the space group from the reference file and set this as the space group at the start of multiplex processing (and check it doesn't clash with any user specified space group). Then we won't need to specify it here.

Reply:
Think this should be updated as well - let me know if you think of a better way of doing this or anything else to add.